src/screens/Dashboard/ThreadList/ThreadItem/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { FC, memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Text } from '@metacraft/ui';
+import CommentInput from 'components/CommentInput';
+import BellIcon from 'components/icons/feather/Bell';
+import PinIcon from 'components/icons/feather/Pin';
```

Comment:
oops, my mistake -- we could not use `aliased` import.
It must be relative path, like `../../../../components/icons/feather/Bell`.

Reply:
i've fixed, pls check again